spec/spec_helper.rb

```diff
@@ -22,7 +22,8 @@
 RSpec.configure do |c|
   default_facts = {
     puppetversion: Puppet.version,
-    facterversion: Facter.version
+    facterversion: Facter.version,
+    service_provider: 'none'
```

Comment:
this is managed by our modulesync_config. Can you please put it into spec/default_module_facts.yml?

Also: Why none? Is this a value that any system reports? I expected systemd or sysv or something similar.

Reply:
Fixed by removing it 👍